File: market/src/main/java/wanted/market/domain/member/repository/MemberRepository.java
package wanted.market.domain.member.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import wanted.market.domain.member.repository.entity.Member;

import java.util.Optional;

public interface MemberRepository extends JpaRepository<Member, Long> {
    Optional<Member> findMemberByEmail(String email);
}

File: src/main/kotlin/wanted/market/api/member/repository/MemberRepository.kt
package wanted.market.api.member.repository

import org.springframework.data.jpa.repository.JpaRepository
import wanted.market.api.member.domain.entity.Member

interface MemberRepository : JpaRepository<Member, Long> {
    fun findByUserIdAndPassword(userId: String, password: String): Member?
    fun findByUserId(userId: String): Member?
}